lib/presentation/my_plan/components/nondata.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+
+import '../../../utils/styles/app_color.dart';
+import '../../../utils/styles/app_text_style.dart';
+import '../../components/wide_button.dart';
+
+class Nondata extends StatelessWidget {
```

Comment:
名前が少しわかりづらいような気がするので、NonDataViewやEmptyDataViewなどにしましょう

```suggestion
class EmptyDataView extends StatelessWidget {
```